tron/core/actionrun.py

```diff
@@ -254,12 +274,18 @@ def start(self):
         return True
 
     def stop(self):
+        if self.retries is not None:
+            self.retries = -1
+
         stop_command = self.action_runner.build_stop_action_command(
             self.id, 'terminate',
         )
         self.node.submit_command(stop_command)
 
     def kill(self):
+        if self.retries is not None:
+            self.retries = -1
```

Comment:
I hope this is right. Seems like if an operator is giving this signal, they don't literally mean "kill this one try attempt", they probably actually mean "kill this attempt and cancel any retries"

Reply:
yeah, worst case they will have to also manually start the job again. it's still better than knowing you have to run kill command X times.